fe/src/main/java/org/apache/doris/catalog/Replica.java

```diff
@@ -318,12 +318,17 @@ private void updateReplicaInfo(long newVersion, long newVersionHash,
             long lastFailedVersion, long lastFailedVersionHash, 
             long lastSuccessVersion, long lastSuccessVersionHash, 
             long newDataSize, long newRowCount) {
-        LOG.debug("before update: {}", this.toString());
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("before update: {}", this.toString());
+        }
 
         if (newVersion < this.version) {
-            // yiguolei: could not find any reason why new version less than this.version should run???
-            LOG.warn("replica {} on backend {}'s new version {} is lower than meta version {}",
-                    id, backendId, newVersion, this.version);
+            // This case means that replica meta version has been updated by ReportHandler before
```

Comment:
why the replica meta version been updated by ReportHandler before?
this is abnormal, I think using waring is better

Reply:
@yangzhg 
for example, now, the publish version daemon send publis verison task(2 3 4 5 6) to one be, and the be finish all publish version task, its replica version is 6 now, but fe need wait other be finish all publish version, now the replica version in fe may be 4, I think it is the common case that be replica version higher than fe replica version, when ReportHandler sync tablet. 
https://github.com/apache/incubator-doris/blob/14ac49dde557b54e9578d91f9dba7e148e5c975b/fe/src/main/java/org/apache/doris/master/ReportHandler.java#L473-L477

Reply:
Good explaination. I suggest to add this directly to the comment.